packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -161,14 +162,6 @@ $-chevron-expand-more: map-merge(
   }
 }
 
-.usa-footer__primary-content {
-  border-top: 1px solid color("base-light");
-
-  @include at-media("mobile-lg") {
-    border: none;
-  }
-}
-
```

Comment:
**Issue (blocking):** While this fixed default and slim variants, we've lost the border on the big footer accordion buttons:

| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/7adf6a67-80b8-4b78-b076-a4255ac2296e) | ![image](https://github.com/user-attachments/assets/7f0779b1-06cb-4af6-9c58-111741b4a1a1) | 

**Suggestion:** Since the markup varies between variants we might just want to store this border as a variable and reapply it for the `.usa-footer__primary-link-button` class:

```scss
$primary-content-border: 1px solid color("base-light")

.usa-footer__primary-link {
   border-top: $primary-content-border;
    ...
    &-button {
    border: 0;
    border-top: $primary-content-border;
    ...
    };
};
```

If there's a cleaner way to accomplish this I'm open to other suggestions as well

Reply:
alternatively, wouldn't elevating the border reset to the link itself, and then removing the border reset from the button work?
```scss
.usa-footer__primary-link {
  border: 0;
  border-top: 1px solid color("base-light");
  /* ... */
  &--button {
    /* remove border: 0 */
  }
}```

Reply:
Good idea! I hadn't considered that. If that works without regression then we can take that route!

Reply:
From my testing, this seems to fix the issue, and IMHO it is a little cleaner, as it keeps the border definition in one place for all footer primary links, regardless of if they're truly links or instead collapsible buttons. Let me know your thoughts, and I can submit the change @mahoneycm

Reply:
Sounds good, I think we commented at the same time. Will make the change.